index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
+    integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="header">
+    <div class="logo">
+      <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt=""
+        id="header-img">
+    </div>
+    <div id="nav-bar">
+      <ul>
+        <a href="#features" class="nav-link">
+          <li>features</li>
+        </a>
+        <a href="#how" class="nav-link">
+          <li>how it works</li>
+        </a>
+        <a href="#cards" class="nav-link">
+          <li>Pricing</li>
+        </a>
+      </ul>
+    </div>
+  </div>
+
```

Comment:
[REQUIRED] Good job so far! Please change the <div id="header"> tag to header tag  and <div id="nav-bar" to nav tag🙏🏼 The nav and header tags  are  for semantics reasons, and the right position for convention and user experience for mobile devices 👍🏼

Reply:
- [ ] Nice job :clap: please give more explanation about `semantic` tags and why you use `semantic` tags so the student understands easily the reason for your required changes :slightly_smiling_face:

Reply:
Semantic HTML tags  uses  HTML markup to reinforce the semantics, or meaning, of the information in web pages and web applications rather than merely to define its presentation or look.

Reply:
Well, the last reviewer asked you to explain semantics a bit but now in the latest comment from your side, I can see the explanation of semantics without describing what the user needs to change.

### Why this happened and what to do now?

Well, actually what the last Code Reviewer meant was to recreate the comment or edit it by describing more clearly about semantics not only describe something related to semantics. So, consider kind of taking both of your comments making a good sentence from both of them, and making another comment asking for a change with a descriptive reason, that's it. 💯

Reply:
[REQUIRED] Good job so far! Please change the

tag to header tag and <div id="nav-bar" to nav tag🙏🏼 The nav and header tags are for semantics reasons. Which reinforces the  meaning of the information in web pages and web applications rather than merely to define its presentation .These tags are also  for convention and user experience for mobile devices 👍🏼